default.cfg

```diff
@@ -43,6 +43,7 @@ music_volume		8
 parallelism_patch_columns		3
 parallelism_realcolor_tint		8
 scale_melt		true
+semi_translucent_fuzz	true

```

Comment:
Almost yes.
It is inspired by the unique effect that Maes added by default in 32-bit mode.
But the effect was buggy: when you see alpha-spectre, say, after you shoot a rocket, you could see frames of rocket explosion behind spectre instead of real background. This is similar to Hall of Mirrors effect.

The reason of that is the fact, that we do not blend alpha with background-rendered picture like it is possible in OpenGL modes, instread we _overwrite_ the existing background picture pixels with pixels that have some alpha. Because of buffering, the the new frame will only close the previous frame by a percent of alpha at the positions of these pixels. Thus, we having a Spectre-of-Mirror that way.

What I did:
I use it like normal, like in 24 bit - I add pixels with respect to colormap #6. But I add alpha that way, so the next frame it will take a previous frame of FUZZ instead of previos frame of the general background. So I have the same effect, but it HOMing only itself, i.e. you cannot see rocket explosion, but you can see as FUZZY bounds leave some percent of trail of prevous frames of FUZZ.

There is no reason to make if false by default, because by default we use Indexed mode which is like vanilla. And this option is totally ignored in any mode except for 32bit with alpha.